src/test/resources/org/prebid/server/it/storedimps/test-genericAlias-stored-video-cache-update.json

```diff
@@ -18,7 +18,7 @@
     ]
   },
   "ext": {
-    "rubicon": {
+    "genericAlias": {
```

Comment:
We don't need rubicon properties
just Just some `"property2":"value2"`

Reply:
Done